toolkit-docker-image/ReadMe.md

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+Creating MGW-ToolKit Image
+Note that the root directory is called <dockerfile-home>
+
+1. Create a directory called "files" in <dockerfile-home>
```

Comment:
Check the formatting. When u open this file in git hub it shows only Create a directory called "files" in. <dockerfile-home> gets removed due to the formatting issues. Please fix other places as well.